### DISCUSSION TITLE
When to expect compat commits for new kernels?

### DISCUSSION THREAD

**Discussion by darkbasic** [openzfs/zfs#15667] [Q&A] [answered]
Hi,
Linux 6.7 is already at rc5 and I was looking for compat commits in the master branch to backport to 2.2. When do new kernels get usually supported? For 6.6 it was around rc4/rc5. I still didn't try the dkms against 6.7, AFAIK it may not even need modifications but usually the maximum supported kernel version gets bumped anyway.

**Comment by robn** [comment, marked as answer]
Not done yet. At least, #15582 needs resolving.

**Comment by robn** [comment]
#15681 might take care of this. I'd appreciate review and testing.
